Content.Server/Guardian/GuardianHostComponent.cs

```diff
@@ -0,0 +1,24 @@
+using Robust.Shared.Containers;
+using Robust.Shared.GameObjects;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.Guardian
+{
+    /// <summary>
+    /// Given to guardian users upon establishing a guardian link with the entity
+    /// </summary>
+    [RegisterComponent]
+    [ComponentProtoName("GuardianHost")]
+    public sealed class GuardianHostComponent : Component
+    {
+        /// <summary>
+        /// Guardian hosted within the component
+        /// </summary>
+        public EntityUid HostedGuardian;
```

Comment:
This should probably just be nullable, instead of defaulting to invalid and checking `IsValid()`.

Reply:
Actually given that this is supposed to be set whenever the component is added, maybe it shouldn't be nullable, but the `IsValid()` checks shouldn't be needed either.

Reply:
It's not necessarily set tho, GuardianHost can be added at any time to any entity.